config/initializers/edition_services.rb

```diff
@@ -17,6 +17,14 @@
     end
   end
 
+  coordinator.subscribe(/^(force_publish|publish)$/) do |_event, edition, options|
+    edition.attachables.flat_map(&:attachments).each do |attachment|
+      ServiceListeners::AttachmentLinkHeaderUpdater
+        .new(attachment)
+        .update!
+    end
+  end
```

Comment:
@floehopper: Is this what you meant by being more selective about the events we subscribe to?

Reply:
@chrisroos All I can see is this bit of the diff:

```diff
+        .new(attachment)
+        .update!
+    end
+  end
```

So I can't really tell, but I meant that the updater would only be subscribed to a subset of workflow events.